fractal-puppy/Engine
# Unequip working for every type of item

[Bug #971 ](https://app.hacknplan.com/p/85404/kanban?categoryId=8&boardId=212215&taskId=971&tabId=basicinfo)

Now, unequipping an item doesn't unequip the weapon as well.

**To test:**
1. Open the graveyard scene.
2. Hit play.
3. Pick up the weapon and the helmet, equip the weapon and then equip the helmet.
4. See how now the weapon doesn't disappear from player hands when the helmet is equipped.
5. Unequip the helmet.
6. See how now the weapon doesn't disappear from player hands when the helmet is unequipped.
